actions/cloud.tsx

```diff
@@ -69,3 +74,14 @@ export function completeStripeAddPaymentMethod(stripe: Stripe, billingDetails: B
         return true;
     };
 }
+
+export function updateCloudSubscription(productId: string) {
```

Comment:
To be consistent with the actual `client4` action

```suggestion
export function subscribeCloudSubscription(productId: string) {
```